install-extensions.sh

```diff
@@ -26,6 +26,13 @@ function link_extension() {
     done
 }
 
+# Step 3: restarting Klipper
+function restart_klipper()
+{
+    echo "Restarting Klipper..."
+    sudo systemctl restart klipper
+}
```

Comment:
Before taking this, let me do some research into Moonraker. Their documentation seems to suggest that the `managed_services` setting should be used to let Moonraker know which services to restart. Therefore, there shouldn't be a need to restarted it as part of the script.

Reply:
The problem is when I was testing out the install script, I need to do it a couple of times to fix the issues I found.   I was not using moonraker to do that.

Moonraker will only come into play when you do a release of your Github repo which add a release tag to your GitHub rep and then moonraker can update if you make changes down the road.

Having the restart in the file does not hurt it, even if you have the moonraker option to bind Klipper to your extension, but it sure comes in handy for testing

Reply:
For some information, having Klipper restart does not effect the ability for moonraker to update automatically.  moonraker will just delay the Klipper restart when it detects it during and update. That way the install occurs and the restart is placed at the end